Tests/js/views/ez-searchview.html

```diff
@@ -0,0 +1,88 @@
+<!doctype html>
+<html>
+<head>
+    <title>eZ Search view tests</title>
+</head>
+<body>
+
+<div class="container"></div>
+
+<script type="text/x-handlebars-template" id="searchview-ez-template">
+    <div class="ez-searchview-content">
+        <div class="ez-searchview-search-form">
+            <form class="ez-search-form pure-form">
+                <input class="ez-search-form-input" required type="text" value="">
+                <button type="submit" class="pure-button ez-button">Search</button>
+            </form>
+            <div class="ez-searchlist-content"></div>
+        </div>
+    </div>
+</script>
+
+<script type="text/javascript" src="../../../Resources/public/vendors/yui3/build/yui/yui.js"></script>
+<script type="text/javascript" src="./assets/ez-searchview-tests.js"></script>
+<script>
+    var filter = (window.location.search.match(/[?&]filter=([^&]+)/) || [])[1] || 'raw',
+            loaderFilter;
+    if (filter == 'coverage'){
+        loaderFilter = {
+            searchExp : "/Resources/public/js/",
+            replaceStr: "/Tests/instrument/Resources/public/js/"
+        };
+    } else {
+        loaderFilter = filter;
+    }
+
+    YUI({
+        coverage: ['ez-searchview'],
+        filter: loaderFilter,
+        modules: {
+            "ez-searchview": {
+                requires: [
+                    'ez-templatebasedview',
+                    'ez-searchlistview',
+                    'node-style',
+                    'ez-loadmorepagination',
+                    'event-valuechange',
+                ],
+                fullpath: "../../../Resources/public/js/views/ez-searchview.js"
+            },
+            "ez-searchlistview": {
+                requires: ['ez-searchplugin', 'ez-subitemlistitemview', 'ez-loadmorepagination'],
+                fullpath: "../../../Resources/public/js/views/ez-searchlistview.js"
+            },
+            "ez-loadmorepagination": {
+                requires: ['view', 'event-tap'],
+                fullpath: "../../../../Resources/public/js/extensions/ez-loadmorepagination.js"
+            },
+            "ez-subitemlistitemview": {
+                requires: ['ez-templatebasedview', 'event-tap', 'event-valuechange', 'template'],
+                fullpath: "../../../../Resources/public/js/views/subitem/ez-subitemlistitemview.js"
+            },
+            "ez-searchplugin": {
```

Comment:
same remark as in the search list view HTML file, this dependency has nothing to do here, `ez-searchplugin` is not a dependency of ez-searchlistview